test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2372,25 +2281,25 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
             )
 
             $testCases2 = @(
-                @{ Test = @{IntendedProtocol = 'Tls';   ActualProtocol = 'Tls12'}; Pending = $false }
-                @{ Test = @{IntendedProtocol = 'Tls';   ActualProtocol = 'Tls11'}; Pending = $false }
+                @{ Test = @{IntendedProtocol = 'Tls'; ActualProtocol = 'Tls12'}; Pending = $false }
+                @{ Test = @{IntendedProtocol = 'Tls'; ActualProtocol = 'Tls11'}; Pending = $false }
                 @{ Test = @{IntendedProtocol = 'Tls11'; ActualProtocol = 'Tls12'}; Pending = $false }
                 @{ Test = @{IntendedProtocol = 'Tls12'; ActualProtocol = 'Tls11'}; Pending = $false }
                 @{ Test = @{IntendedProtocol = 'Tls11'; ActualProtocol = 'Tls'}; Pending = $false }
                 @{ Test = @{IntendedProtocol = 'Tls12'; ActualProtocol = 'Tls'}; Pending = $false }
                 # macOS does not support multiple SslProtocols
-                @{ Test = @{IntendedProtocol = 'Tls11, Tls12';   ActualProtocol = 'Tls'}; Pending = $IsMacOS }
-                @{ Test = @{IntendedProtocol = 'Tls, Tls12';   ActualProtocol = 'Tls11'}; Pending = $IsMacOS }
-                @{ Test = @{IntendedProtocol = 'Tls, Tls11';   ActualProtocol = 'Tls12'}; Pending = $IsMacOS }
+                @{ Test = @{IntendedProtocol = 'Tls11, Tls12'; ActualProtocol = 'Tls'}; Pending = $IsMacOS }
+                @{ Test = @{IntendedProtocol = 'Tls, Tls12'; ActualProtocol = 'Tls11'}; Pending = $IsMacOS }
+                @{ Test = @{IntendedProtocol = 'Tls, Tls11'; ActualProtocol = 'Tls12'}; Pending = $IsMacOS }
```

Comment:
Original vertical formatting is better.

Reply:
@iSazonov somehow I missed this comment. 

I agree that the original alignment is better, but, it was also inconsistent with how it was done in other areas of this file.  I haven't found a way in VSCode to keep this kind of white space alignment or to auto-format single line hashtables like this.  The only decent alignment option is the one that puts each key/value on a separate line and aligns the `=`'s. for a single hash table, that improves readability, but for an array of hashes it actually makes it less readable. ☹️

Reply:
Can we request the feature?

Reply:
Probably... I wouldn't know how to word it myself. I have an idea of how it should look, but not a good enough definition to even pseudo code it.

Reply:
I believe the sample that we discuss is good for the request to show that we want.
Or perhaps there are directives to disable this unwanted formatting?

Reply:
If there is a way to disable it, I certainly haven't found it. I make heavy use of this style for complex logic (silly example):

```powershell
if 
(
    $Kizumongatari.Episodes.Count -lt $Bakemonogatari.  Episodes.Count -and
    $Kizumongatari.Episodes.Count -lt $Nekomonogatari.  Episodes.Count -and
    $Kizumongatari.Episodes.Count -lt $Nisemonogatari.  Episodes.Count -and
    $Kizumongatari.Episodes.Count -lt $SecondSeason.    Episodes.Count -and
    $Kizumongatari.Episodes.Count -lt $Tsukimonogatari. Episodes.Count -and
    $Kizumongatari.Episodes.Count -lt $Owarimonogatari. Episodes.Count -and
    $Kizumongatari.Episodes.Count -lt $Koyomimonogatari.Episodes.Count 
)
{
    'Kizumonogatari has the lowest episode count.'
}
```

And when I started formatting with VScode I lost much of it in my projects. I spent a few days toying with the various settings but it seems like I'd have to just not use any kind of automated formatting if I want to keep this code style.

> I believe the sample that we discuss is good for the request to show that we want.

It's one thing to show it, it but it's a whole other thing to define it well enough for someone to implement. Going to the VS Code or PowerShell extension repos with a vague "I would like a formatting option that looks like this" wont be very productive, IMO.